.gitignore

```diff
@@ -61,3 +61,10 @@
 .idea
 .vscode/ipch
 /newIDE/app/src/UI/Theme/**/*ThemeVariables.*
+.DS_Store
+.DS_Store?
+._*
+.Spotlight-V100
```

Comment:
I would rather avoid these things that could encompass too much files.

Reply:
Okay